### PR TITLE
Use extension registration for MediaWiki extension credits

### PR DESCRIPTION
**Pull request by filbertkm**
also fixes broken license link on Special:Version

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
If possible, I think it'd be nice to have this file be named `mediawiki.json` rather than `extension.json`, so it's more clear what it is about.

File: WikibaseDataModel.php [hunk 1]
```diff
@@ -7,16 +7,9 @@
...
-	// Do not initialize more than once.
-	return 1;
-}
-
-define( 'WIKIBASE_DATAMODEL_VERSION', '4.4.0 alpha' );
-
 if ( defined( 'MEDIAWIKI' ) ) {
 	call_user_func( function() {
-		require_once __DIR__ . '/WikibaseDataModel.mw.php';
+		wfLoadExtension ( 'WikibaseDataModel', __DIR__ . '/extension.json' );
```
Comment (thiemowmde) [inline WikibaseDataModel.php]: There shouldn't be a space after the function name.

File: WikibaseDataModel.php [hunk 2]
```diff
@@ -7,16 +7,9 @@
...
 
-if ( defined( 'WIKIBASE_DATAMODEL_VERSION' ) ) {
-	// Do not initialize more than once.
-	return 1;
-}
-
-define( 'WIKIBASE_DATAMODEL_VERSION', '4.4.0 alpha' );
-
 if ( defined( 'MEDIAWIKI' ) ) {
 	call_user_func( function() {
```
Comment (thiemowmde) [inline WikibaseDataModel.php]: Is this call_user_fund still necessary? I don't think so.

Comment (filbertkm) [inline WikibaseDataModel.php]: removed it

File: WikibaseDataModel.php (hunk 1 above)
Comment (filbertkm) [inline WikibaseDataModel.php]: done

**Comment by thiemowmde** [comment]
+2 so far from my side. I want to wait for the result of the other discussions.

**Pull request merged**